test/baseline.spec-cpp.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+
+#TODO(Jesse): Maybe make another results file for the failures?
+
+output_filename=./baseline.spec-cpp.results
+
+[ -f $output_filname ] && rm $output_filename
+
+for filename in _tmp/spec/cpp/*.tsv; do
+
+  passes="$(cat "$filename" | grep "osh_\.cc\spass" | grep -P -o "\d+")"
+  for test_number in $passes; do
+    echo -n $test_number >> $output_filename
+    echo -n " "          >> $output_filename
+    echo -n $filename    >> $output_filename
+    echo ""              >> $output_filename
```

Comment:
how about just:

echo "test_number $filename" >> $output_filename

Reply:
Haha, yeah, not exactly sure why I took the long way round ..